misk-hibernate/src/test/kotlin/misk/hibernate/ReflectionQueryFactoryTest.kt

```diff
@@ -48,6 +50,9 @@ class ReflectionQueryFactoryTest {
       session.save(DbMovie(m98.name, m98.releaseDate))
       session.save(DbMovie(m99.name, m99.releaseDate))
 
+      OperatorsMovieQuery::class.functions
```

Comment:
is this needed?

Reply:
Whoops, was using that while building and learning about ReflectionQueryFactory, not needed now.